src/popup/fxa.js

```diff
@@ -38,13 +38,14 @@ async function init() {
 
 function setupAccountMenu(user) {
   if (user) {
-    if (document.getElementById("email")) {
-      document.getElementById("email").innerText = user.email;
+    const emailElement = document.getElementById("email");
+    if (emailElement) {
+      emailElement.innerText = user.email;
 
-      if (user.avatarDefault) {
-        document.getElementById("avatar").style.backgroundImage = `url("/icons/avatar.svg")`;
-      } else if (user.avatar) {
+      if (user.avatar) {
```

Comment:
Flipped this condition because I'm not returning `avatarDefault` any more, as you'll see when you scroll down...